<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Rohits-MacBook-Air.local" failures="1" tests="7" name="Selenium.Cucumber.RunCucumberTest" time="107.449" errors="0" timestamp="2025-01-30T12:11:24 IST" skipped="0">
  <testcase classname="Selenium.Cucumber.RunCucumberTest" name="runScenario" time="14.021"/>
  <testcase classname="Selenium.Cucumber.RunCucumberTest" name="runScenario" time="13.617"/>
  <testcase classname="Selenium.Cucumber.RunCucumberTest" name="runScenario" time="13.178"/>
  <testcase classname="Selenium.Cucumber.RunCucumberTest" name="runScenario" time="14.130"/>
  <testcase classname="Selenium.Cucumber.RunCucumberTest" name="runScenario" time="17.132"/>
  <testcase classname="Selenium.Cucumber.RunCucumberTest" name="runScenario" time="20.539">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at Selenium.Cucumber.TestBase.after(TestBase.java:120)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="Selenium.Cucumber.RunCucumberTest" name="runScenario" time="14.832"/>
</testsuite> <!-- Selenium.Cucumber.RunCucumberTest -->
